### PR TITLE
Change the gradle build to use the gogradle plugin

### PR DESCRIPTION
**Pull request by houshengbo**
This PR applies the gogradle plugin to build the binaries of openwhisk
wskdeploy.

Partially-closes: apache/incubator-openwhisk-release#9

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
 		},
 		{
 			"ImportPath": "github.com/mattn/go-isatty",
-			"Rev": "66b8e73f3f5cda9f96b69efd03dd3d7fc4a5cdb8"
+			"Rev": "a5cdd64afdee435007ee3e9f6ed4684af949d568"
```
Comment (mrutkows) [inline Godeps/Godeps.json]: are there any other deps we should be updating at this time to match build.gradle versions?

File: build.gradle [hunk 2]
```diff
@@ -43,129 +93,217 @@ rat {
...
+        }
+    }
+
+    doLast {
+        mkdir('./build')
+        file('./build/content.json').text = groovy.json.JsonOutput.toJson(["wskdeploy": content])
+    }
+}
+
+task releaseBinaries(type: Tar, dependsOn: [individualArchives, index]) {
```
Comment (mrutkows) [inline build.gradle]: Have we run a tagged release PR to test?

Comment (houshengbo) [reply build.gradle]: well we have to merge it and test :-(

File: Godeps/Godeps.json (hunk 1 above)
Comment (houshengbo) [reply Godeps/Godeps.json]: no this is the only one i found out to change.

**Pull request merged**